packages/fleather/lib/src/widgets/editable_text_block.dart

```diff
@@ -344,14 +343,12 @@ class _BulletPoint extends StatelessWidget {
 }
 
 class _CheckboxPoint extends StatefulWidget {
```

Comment:
Could you make it stateless? Why did we make it stateful in the first place?! 😀

Reply:
No idea! 😀